modules/apps/27-interchain-accounts/host/types/params.go

```diff
@@ -0,0 +1,59 @@
+package types
+
+import (
+	"fmt"
+
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+const (
+	// DefaultHostEnabled is the default value for the host param (set to true)
+	DefaultHostEnabled = true
```

Comment:
So host is enabled by default and controller must be enabled via gov proposal?

Reply:
Turns out ics20 is enabled by default so I copied that. Both host/controller are currently enabled by default. We could set this to false. We will need to add some code into our tests so that we don't use the default value (if we set it to false)